ground/gcs/src/plugins/config/configplugin.cpp

```diff
@@ -75,9 +75,9 @@ bool ConfigPlugin::initialize(const QStringList &args, QString *errMsg)
     connect(telMngr, &TelemetryManager::connected, this, &ConfigPlugin::onAutopilotConnect);
     connect(telMngr, &TelemetryManager::disconnected, this, &ConfigPlugin::onAutopilotDisconnect);
 
-    // And check whether by any chance we are not already connected
```

Comment:
We don't have a device list until after all the plugins are loaded: https://github.com/d-ronin/dRonin/blob/next/ground/gcs/src/plugins/coreplugin/connectionmanager.cpp#L466